include/drivers/rtc.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_RTC_H_
+#define ZEPHYR_INCLUDE_DRIVERS_RTC_H_
+
+/**
+ * @file
+ * @brief Public RTC driver APIs
+ */
+
+#include <zephyr/types.h>
+#include <time.h>
+#include <device.h>
+
+typedef int (*rtc_api_gettime)(struct device *dev, struct timespec *tp);
```

Comment:
we usually add '_t' in the end of typedef-ed names

Reply:
I fix it!